### ISSUE TITLE
Tales of Hearts no longer working.

### ISSUE THREAD

**Issue by Messybirb1** [Vita3K/Vita3K#1718]
Tales of Hearts has suddenly stopping working on all builds even previous builds that used to run it no longer runs Tales of Hearts.
I tried deleting everything, but it just no longer works. 

![heartscrash](https://user-images.githubusercontent.com/103961657/167883683-7eafa7e1-d23c-4f39-8907-74b957aaa986.PNG)

**Comment by Macdu** [comment]
Indeed, pr #1712 led to some game like uncharted or tales of Hearts R not booting anymore. Until pr #1676  is merged, you can still try the game by going to Configuration -> Settings -> Core, choose Auto & Manual and then enable libhttp.
